client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -601,6 +609,43 @@
         };
     }
 
+    PackagingEmbeddedDirective.$inject = ['packages'];
+    function PackagingEmbeddedDirective(packages) {
+        return {
+            templateUrl: 'scripts/superdesk-packaging/views/packaging.html',
+            require: '^sdAuthoringContainer',
+            scope: {
+                item: '='
+            },
+            link: function(scope, elem, attrs, authoringCtrl) {
+                scope.$watch('item', function(item) {
+                    if (item && item.lockIt){
+                        scope.lock();
+                    } else {
+                        scope.origItem = null;
+                        scope.$applyAsync(function() {
```

Comment:
Ugh, this sounds quite hacky, although I now understand the reason. Is there perhaps a good way to achieve the same in a different way? For example, the `sd-authoring` directive could expose a controller with `reset()` method which can be used by other directives to reset it.

Just thinking aloud, I'm not familiar with all the implementation details...